graveler/committed/sstable/iterator.go

```diff
@@ -49,18 +45,17 @@ func (iter *Iterator) Next() bool {
 		return false
 	}
 
-	iter.valParsed, iter.err = iter.ser.DeserializeValue(iter.currValue)
-	return iter.err == nil
+	return true
 }
 
-func (iter *Iterator) Value() *graveler.ValueRecord {
+func (iter *Iterator) Value() *committed.Record {
 	if iter.currKey == nil || iter.err != nil || iter.postSeek {
```

Comment:
if next() was not called after SeekGE - you return nil. It indicates a bug.  Maybe an error should be posted as well

Reply:
I follow the interface as we decided upon it and documented. If we want to change the behaviour we need to do so for all iterators, not just here.